packages/react/src/components/DropdownMenu/DropdownMenuGroup.tsx

```diff
@@ -25,7 +25,7 @@ export const DropdownMenuGroup = forwardRef<
         {...(heading ? { 'aria-labelledby': headingId } : {})}
         ref={ref}
         role='group'
-        className={'ds-dropdownmenu__section'}
+        className={'ds-dropdownmenu__group'}
```

Comment:
should expose `className={cl('ds-dropdownmenu', className)}` here too? ☺️

Reply:
We try to put their `className` and `style` on the outermost element, which in this case is the `li` element

Reply:
oh, sorry - did not see this! carry on :D